chapter_linear-regression/linear-regression-concise.md

```diff
@@ -60,6 +60,16 @@ import numpy as np
 import tensorflow as tf
 ```
 
+```{.python .input  n=1}
+%%tab jax
+from d2l import jax as d2l
+import jax
+from jax import numpy as jnp
+from jax import grad
+from flax import linen as nn
+import optax
+```
```

Comment:
Is that something we do everywhere? Because alphabetical imports are inconsistent across d2l even outside of JAX. We should then fix everything.

Also, I thought it didn't matter. We use `isort` to format imports anyway https://github.com/d2l-ai/d2l-book/pull/27/commits/96ab562095f1d613700213625e7abee4f164e949

Reply:
Yes that's for backup in the final presentation. A benefit of sorting them in each markdown file is that it's easy to avoid duplications

Reply:
cool, I'll update that. let's be consistent about that everywhere then. There tonnes of code blocks which are not following the alphabetical import style